x-pack/qa/security-tls-basic/src/test/resources/rest-api-spec/test/10_tls_basic.yml

```diff
@@ -0,0 +1,27 @@
+setup:
```

Comment:
My 2 cents is that a rest test isn't the right thing here. They are harder to debug and unless we'd want these to be run by language clients, I think a Java test is a better alternative.